### PR TITLE
Ops/eject help

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

**Review by chinook25 — APPROVED** [review]

File: app/ts/InlineHelp/lexicon.ts [hunk 1]
```diff
@@ -1,136 +1,136 @@
-import IHelpInfo from './IHelpInfo';
+import {IHelpInfo} from 'help-popup';
 
 export const lexicon: Record<string, IHelpInfo> = {
   'add-subproblem': {
     title: 'Adding a new problem',
     text:
       'Here you can define a new problem as a subset of all evidence (by excluding criteria and alternatives), and with different scale ranges.',
-    link: '/manual.html#mcda-problem-creation'
+    link: 'mcdaHost/manual.html#mcda-problem-creation'
```
Comment (DanielReid) [inline app/ts/InlineHelp/lexicon.ts:8]: prefer to go the other direction and only have anchor here, and move host plus path to `HelpContext` configuration

File: webpack.common.js [hunk 2]
```diff
@@ -28,10 +29,19 @@ let config = {
 
   module: {
     rules: [
+      {
```
Comment (DanielReid) [inline webpack.common.js:32]: brute-force approach not ideal as discussed

File: package.json [hunk 3]
```diff
@@ -111,7 +111,7 @@
     "font-awesome": "~4.7.0",
     "foundation-sites": "~6.5.3",
     "geckodriver": "^1.19.0",
-    "help-popup": "drugis/help-popup#0.5.1",
+    "help-popup": "drugis/help-popup#1.1.0",
```
Comment (DanielReid) [inline package.json:114]: I feel we could have retagged the new version as the actual 1.0.0 (not like anyone else was using that version) but eh

**Review by DanielReid — APPROVED** [review]
much nicer :)

**Review by chinook25 — APPROVED** [review]

**Pull request merged**